src/ui/Settings.qml

```diff
@@ -288,6 +305,65 @@ Page {
                     }
 
                 } // GroupBox (addressBook setting)
+                
+        GroupBox{
+            Layout.fillWidth: true
+            title: qsTr("Log level")
+
+            RowLayout{
+
+                anchors.fill: parent
+                TextField {
+                    id: textFieldLogLevel
+
+                    selectByMouse: true
+
+                    placeholderText: qsTr("Log level")
+                    onTextChanged: {
+                        updateFooterButtonsStatus();
+                    }
+                }
+            }
+        }
+
+        GroupBox{
+            Layout.fillWidth: true
+            title: qsTr("Log output")
+
+            RowLayout{
+
+                anchors.fill: parent
+                TextField {
+                    id: textFieldLogOutput
+
+                    selectByMouse: true
+
+                    placeholderText: qsTr("Log output")
+                    onTextChanged: {
+                        updateFooterButtonsStatus();
+                    }
+                }
+            }
+        }
+        // RowLayout {
```

Comment:
remove if not needed

Reply:
That code is for the inital Combo Box that I made, I left it commented for @stdevCrow to fix it

Reply:
Code removed